native/acter/src/api/stream.rs

```diff
@@ -457,7 +458,7 @@ impl TimelineStream {
                     bail!("No permission to send message in this room");
                 }
 
-                timeline.cancel_send(&transaction_id).await;
```

Comment:
why does this not get a `?`?

Reply:
`Timeline::cancel_send()` doesn't return `Result<>`.